packages/inventory-compliance/src/ComplianceRemediationButton.js

```diff
@@ -44,10 +40,10 @@ class ComplianceRemediationButton extends React.Component {
         const result = { systems: [], issues: [] };
         allSystems.forEach(async (system) => {
             result.systems.push(system.id);
-            if (selectedRules) {
+            if (selectedRules.length !== 0) {
                 result.issues.push(this.rulesWithRemediations(selectedRules, system.id));
             } else {
-                result.issues.push(this.rulesWithRemediations(system.rule_objects_failed, system.id));
+                result.issues.push(this.rulesWithRemediations(system.ruleObjectsFailed, system.id));
```

Comment:
If you change this, you also have to change in SystemRulesTable: `ruleObjectsFailed: [] }] }`